pkg/controllers/dynakube/istio/reconciler.go

```diff
@@ -38,6 +39,39 @@ func NewReconciler(istio *Client) Reconciler {
 	}
 }
 
+func (r *reconciler) ReconcileCSIDriver(ctx context.Context, dynakube *dynatracev1beta2.DynaKube) error {
+	log.Info("reconciling istio components for the CSI driver")
+
+	if dynakube == nil {
+		return errors.New("can't reconcile csi driver of nil dynakube")
+	}
+
+	codeModulesURL := dynakube.Status.CodeModules.ImageID
```

Comment:
`dynakube.CodeModulesImage()` does the same

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/3343/commits/b5566bd5bdf7edc5afefd8d6d43208d1790c1cf8